Package installs tests in sitedir.

Can't help but wonder how much time is wasted working around stupid setuptools
misfeature magic.

diff --git i/setup.py w/setup.py
index 6e7e03f..0394107 100644
--- i/setup.py
+++ w/setup.py
@@ -20,7 +20,7 @@ setup(
     url=github2.__homepage__,
     license='BSD',
     platforms=["any"],
-    packages=find_packages(exclude=['ez_setup']),
+    packages=["github2", ],
     scripts=['github2/bin/github_manage_collaborators'],
     install_requires=[
     ],
